core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -1,7 +1,18 @@
 <ng-container *ngIf="editorState$ | async as es">
+    <dot-device-selector-seo
+        #deviceSelector
+        [apiLink]="es.iframeURL"
+        (selected)="updateCurrentDevice($event)"
+        appendTo="body"
+        data-testId="dot-device-selector"></dot-device-selector-seo>
```

Comment:
Were not doing any extra requests when until this is opened?

Reply:
I think this components fetch the devices on init